pom.xml

```diff
@@ -655,20 +526,6 @@
       <activation>
         <jdk>[1.8,1.9)</jdk>
```

Comment:
I think we should seriously consider making all Fluo projects require Java 8, minimum. Then, we can delete this profile, and promote the configuration up to the main body of this POM.